webnn/gemm.https.any.js

```diff
@@ -0,0 +1,45 @@
+// META: title=test WebNN API gemm operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-gemm
+
+const computeGemm = (builder, resources) => {
+  // MLOperand gemm(MLOperand a, MLOperand b, optional MLGemmOptions options = {});
+  // use 'a' and 'b' for input operands name
+  const [inputOperandA, inputOperandB] = createInputOperands(builder, resources, ['a', 'b']);
+  const operandType = resources.type;
+  const TestTypedArray = TypedArrayDict[operandType];
+  const options = resources.options;
+  const gemmOptions = {};
+  if (options !== undefined) {
```

Comment:
It needs to process **options** of test json file to pass to `gemm(MLOperand a, MLOperand b, optional MLGemmOptions options = {})` when **options.c** is not a scalar, need use new variable `gemmOptions` to save for different builder/context
 ```js
  context = navigator.ml.createContextSync({deviceType});
  //  or 
  // context = await navigator.ml.createContext({deviceType});
  builder = new MLGraphBuilder(context);
```